dist/source/repo-creator/sponsorship.ts

```diff
@@ -0,0 +1,120 @@
+import { RepoCreator } from 'source/services/RepoCreator';
+import { Repository as RepositoryWireModel } from 'source/models/Repository';
+import { OAuth } from 'source/services/OAuth-Auth0';
+import { GitHub } from 'source/services/GitHub';
+import { StripeCheckout, StripeToken } from 'source/services/StripeCheckout';
+import { Router } from 'aurelia-router';
+import { EventAggregator } from 'aurelia-event-aggregator';
+import { autoinject } from 'aurelia-dependency-injection';
+import { computedFrom } from 'aurelia-binding';
+import { Validation } from 'aurelia-validation';
+import underscore from 'underscore';
+
+class Repository {
+	constructor(
+		public owner: string,
+		public name: string,
+		public sponsored: boolean
+	) {}
+
+	equals = (other: Repository): boolean => {
+		return this.owner == other.owner
+			&& this.name == other.name;
+	}
+
+	merge = (other: Repository) => {
+		this.sponsored = this.sponsored || other.sponsored;
+
+	}
+
+}
+
+enum Sections {
+	SPONSORED
+}
+
+@autoinject
+export class Sponsorship {
+	private allTemplates: Repository[] = [];
+	private repositoriesTemplates: Repository[] = [];
+	protected selectedSection: Sections = Sections.SPONSORED;
```

Comment:
Only one `private repositories: Repository[] = [];` is necessary since we don't have to deal with merging.